src/libraries/Microsoft.PowerFx.Interpreter/RecalcEngine.cs

```diff
@@ -132,13 +126,14 @@ public void UpdateVariable(string name, FormulaValue value)
         /// <param name="parameters">parameters for formula. The fields in the parameter record can 
         /// be acecssed as top-level identifiers in the formula.</param>
         /// <param name="options"></param>
+        /// <param name="symbols"></param>
         /// <returns>The formula's result.</returns>
-        public FormulaValue Eval(string expressionText, RecordValue parameters = null, ParserOptions options = null)
+        public FormulaValue Eval(string expressionText, RecordValue parameters = null, ParserOptions options = null, ReadOnlySymbolValues symbols = null)

```

Comment:
>Eval [](http://example.com/codeflow?start=28&length=4)

We shouldn't need change.

ReadOnlySymbolValues supercedes RecordValue, so we shouldn't need both parameters. 

If we have a  ReadOnlySymbolValues, then we can use the overload on line 153.